record-manager-design-list.component.ts

```
@@ -0,0 +1,1062 @@
+    this.service.errormessagesRemove();
+    this.loading = true;
+    const recordManagerDesignId = event.id;
+
+    this.error_msg = [];
+    // Reason for edit check on create
+    this.mainlayout.checkPassworCogIcon('Record_Manager_Person').then((data1: any) => {
+      if (data1) {
+        // JSON['password'] = data1?.password;
+        const reason = data1.reasonForEdit == 'Required' ? reasonForEdit : null;
+        const isActiveRMDesign = this.isActiveRMDesign && this.isActiveRMDesign.length > 0 ? this.isActiveRMDesign : null;
+        const JSON: any = this.jsonService.updateRMPersonJson(this.baseModelTableData.id, this.recordId, this.tableData, recordManagerDesignId, reason, isActiveRMDesign);
```

Biome (1.9.4)

[error] Do not shadow the global "JSON" property.

Consider renaming this variable. It's easy to confuse the origin of variables when they're named after a known global.

(lint/suspicious/noShadowRestrictedNames)